convert2rhel/checks.py

```diff
@@ -75,77 +89,63 @@ def perform_pre_ponr_checks():
 
 
 def check_convert2rhel_latest():
-    """Make sure that we are running the latest version of convert2rhel"""
-    gpg_path = os.path.join(utils.DATA_DIR, "gpg-keys", "RPM-GPG-KEY-redhat-release")
-    ssl_cert_path = os.path.join(utils.DATA_DIR, "redhat-uep.pem")
-    repo_content = (
-        "[convert2rhel]\n"
-        "name=Convert2RHEL Repository\n"
-        "baseurl=https://cdn.redhat.com/content/public/convert2rhel/$releasever/x86_64/os/\n"
-        "gpgcheck=1\n"
-        "enabled=1\n"
-        "sslcacert=%s\n"
-        "gpgkey=file://%s\n" % (ssl_cert_path, gpg_path)
-    )
+    """Make sure that we are running the latest downstream version of convert2rhel"""
+    logger.task("Prepare: Checking if this is the latest version of convert2rhel")
 
```

Comment:
In beginning of file, check if repos can be used (the internet check), if not we skip the check with a warning

```suggestion
    """Make sure that we are running the latest downstream version of convert2rhel"""
    logger.task("Prepare: Checking if this is the latest version of convert2rhel")

    if not system_info.has_internet_access:
        logger.warning("Skipping the check as no internet connection has been detected.")
        return
```

Reply:
I have made this change and will be pushing the changes soon!